src/components/company/compare.tsx

```diff
@@ -10,28 +10,48 @@ import {
   TableRow,
 } from "@mui/material";
 import React from "react";
-import { ICompareResult } from "../../models/company/company.model";
+// import { ICompareResult } from "../../models/company/company.model";
+import { FinancialData } from "../../services/api";
+import { useOutletContext } from "react-router-dom";
+import { IAppContext } from "../../models";
 
 const Compare = () => {
-  const [compareResult, setCompareResult] = React.useState<ICompareResult[]>(
-    []
-  );
-  const [company1, setCompany1] = React.useState<string>("");
-  const [company2, setCompany2] = React.useState<string>("");
+  // const [compareResult, setCompareResult] = React.useState<ICompareResult[]>(
+  //   []
+  // );
+  const { api } = useOutletContext<IAppContext>();
```

Comment:
Hier wordt de api opgehaald.
Zo is het API object dus beschikbaar.
Voorbeeld van hoe dit aan te spreken staat in de readme.